bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -769,6 +769,28 @@ private function addObjectsNode(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('custom_layout')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->arrayNode('definitions')
+                                ->normalizeKeys(false)
+                                    ->prototype('array')
+                                        ->children()
+                                            ->scalarNode('id')->end()
+                                            ->scalarNode('name')->end()
+                                            ->scalarNode('description')->end()
+                                            ->integerNode('creationDate')->end()
+                                            ->integerNode('modificationDate')->end()
+                                            ->integerNode('userOwner')->end()
+                                            ->integerNode('userModification')->end()
```

Comment:
I've encoureted a problem with this being intengerNode, since the custom layout is created in "2 steps" (first "add layout" and then you can edit and save), when first created, it has `null`on `UserModification`
https://github.com/pimcore/pimcore/blob/9cfb411ca31695c64fb164f1f00bf32f7f1df29a/bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php#L322-L328
need to add UserModification default value here or change integerNode